web3/providers/ipc.py

```diff
@@ -136,12 +136,14 @@ def get_default_ipc_path(testnet=False):
 class IPCProvider(JSONBaseProvider):
     _socket = None
 
-    def __init__(self, ipc_path=None, testnet=False, *args, **kwargs):
+    def __init__(self, ipc_path=None, testnet=False, timeout=10,
+                 *args, **kwargs):
```

Comment:
*nitpick*

Our style guide says that if a function doesn't fit, then each argument goes on its own line: https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md#function-signatures

Like:

```py
    def __init__(
        self,
        ipc_path=None,
        ...
        **kwargs):
```

Reply:
Thanks for pointing that out, I'll make the change.